clients/stellar-relay-lib/src/connection/xdr_converter.rs

```diff
@@ -1,10 +1,8 @@
-#![allow(dead_code)] //todo: remove after being tested and implemented
-
 use crate::sdk::types::{
 	AuthenticatedMessage, AuthenticatedMessageV0, HmacSha256Mac, MessageType, StellarMessage,
 };
 use std::fmt::Debug;
-use substrate_stellar_sdk::XdrCodec;
+use substrate_stellar_sdk::{parse_stellar_type, StellarSdkError, XdrCodec};
```

Comment:
`parse_stellar_type` was moved to address
https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/stellar-relay-lib/src/connection/xdr_converter.rs#L40-L41.
See https://github.com/pendulum-chain/substrate-stellar-sdk/pull/25